index.html

```diff
@@ -19,55 +19,38 @@
           shortName:  "jlreq",
 		  noRecTrack: true,
           editors: [
-                {   name:       "<span lang='ja'>千葉 弘幸</span> (Hiroyuki Chiba)",
-                    company:    "Invited Expert",
-                    w3cid:      "40236" },
-                {   name:       "<span lang='ja'>枝本 順三郎</span> (Junsaburo Edamoto)",
+                {   name:       "<span lang='ja'>小林 敏</span> (Toshi Kobayashi)",
                     company:    "Invited Expert",
-                    w3cid:      "40799" },
+                    w3cid:      "40266" },
                 {   name:       "Richard Ishida",
                     company:    "W3C",
                     w3cid:      "3439" },
-                {   name:       "<span lang='ja'>石野 恵一郎</span> (Keiichiro Ishino)",
-                    company:    "Antenna House",
-                    w3cid:      "134219" },
-                {   name:       "<span lang='ja'>加藤 誠一</span> (Seiichi Kato), (<span lang='ja'>第２版</span> 2nd edition)",
-                    company:    "Microsoft",
-                    w3cid:      "45143" },
-                {   name:       "<span lang='ja'>小林 龍生</span> (Tatsuo Kobayashi)",
-                    company:    "1st edition, JustSystems; 2nd edition, Invited Expert",
-                    w3cid:      "37983" },
-                {   name:       "<span lang='ja'>小林 敏</span> (Toshi Kobayashi)",
+                {   name:       "<span lang='ja'>木田 泰夫</span> (Yasuo Kida)",
                     company:    "Invited Expert",
-                    w3cid:      "40266" },
-                {   name:       "Nathaniel McCully, (<span lang='ja'>第２版</span> 2nd edition)",
```

Comment:
should include this @kidayasuo

Reply:
Thank you for spotting this.
Yes, please add Nat. He contributed to the English translations as an editor.